assets/js/source_lv_hooks.js

```diff
@@ -224,6 +224,22 @@ hooks.SourceLogsSearch = {
       }
     }, 250)
   },
-}
+};
+
+/* Listens for a `scrollIntoView` event on the element
+
+  Example:
+
+  <div phx-mounted={JS.dispatch("scrollIntoView")}>
+    ...
+  </div>
+*/
+hooks.ScrollIntoView = {
+  mounted() {
+    this.el.addEventListener("scrollIntoView", (event) => {
+      event.target.scrollIntoView({ behavior: "instant" });
+    });
+  },
+};
```

Comment:
Attempted to avoid a phx-hook here but couldn't make it work. Can revisit, otherwise I'm keen to update LiveView to 1.1 and use colocated hooks.